ValgrindCI/__init__.py

```diff
@@ -55,6 +78,27 @@ def main():
     data.parse(args.xml_file)
     data.set_source_dir(args.source_dir)
 
+    if args.substitute_path:
+        substitute_paths = []
+        for s in args.substitute_path:
+            substitute_paths.append({"from": s.split(":")[0], "to": s.split(":")[1] })
+        data.set_substitute_paths(substitute_paths)
+
+    if args.relativize:
+        prefixes = []
+        for p in args.relativize:
+            prefixes.append(p)
+        data.set_relative_prefixes(prefixes)
```

Comment:
Why are you duplicating the list `args.relativize` in another list `prefixes` ? Wouldn't it be simpler to send `args.relativize` as an argument to `data.set_relative_prefixes()` ?
```suggestion
    if args.relativize:
        data.set_relative_prefixes(args.relativize)
```